src/github/deployment.ts

```diff
@@ -0,0 +1,13 @@
+import transformDeployment from "../transforms/deployment";
+import { Context } from "probot/lib/context";
+
+export default async (context: Context, jiraClient) => {
+	const { data: jiraPayload } = await transformDeployment(context);
```

Comment:
Is there a reason that we call a function that returns an object with a key of `data` that we then completely ignore that data key?  Why doesn't the function just return the jiraPayload object?  No need to faff about.

Reply:
No need, returning only jiraPayload now.